electora_py/api.py

```diff
@@ -1,11 +1,33 @@
-from nucypher.characters.lawful import Bob, Ursula
+from typing import Optional, Dict
+from gql import gql, Client
+from gql.transport.aiohttp import AIOHTTPTransport
+from nucypher_core import ferveo
+
+from nucypher.characters.lawful import Ursula, Bob
 from nucypher.cli.utils import connect_to_blockchain
 from nucypher.utilities.emitters import StdoutEmitter
-from nucypher_core import ferveo
+
+# Networks
 
 _GOERLI_URI = "https://goerli.infura.io/v3/663d60ae0f504f168b362c2bda60f81c"
 _TEACHER_URI = "https://lynx.nucypher.network:9151"
 
+# GraphQL
+
+_ELECTORA_ARWEAVE_TAG = "electora/ballot/uuid"
```

Comment:
This tag format is a suggestion for some more semantic density, but it can be whatever!

Reply:
in order to maintain future compatibility, i think I'm going to go with multiple tags:
```
tags = [{ "name": "ballot_uuid", "value": ballot.uuid },
        { "name": "application", "value": "electora"},
        # vote type,
        # version]
```
That way adding new metadata doesn't invalidate old stuff

Reply:
Opened https://github.com/electora-vote/electora-py/issues/4